game/scripts/npc/abilities/bristleback_quill_spray.txt

```diff
@@ -58,7 +58,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "max_damage"                                      "550.0 550.0 550.0 550.0 1100.0 2200.0"
+        "max_damage"                                      "150.0 275.0 400.0 550.0 825.0 1100.0"
```

Comment:
"I have nerfed the early max damage on Quill Spray. This hardly impacts Bristles fighting ability since to deal the maximum damage he needs to get 5 stacks on the target."

I think it should be: 550.0 550.0 550.0 550.0 825.0 1100.0

This doesn't impact vanilla stats  (something we should try and avoid whenever possible). Was your concern that Bristle was too good at farming bosses with lvl 3 quillspray or something?